src/app/components/transcriptions/approve-transcript/approve-transcript.component.html

```diff
@@ -8,7 +8,7 @@
 
   <app-details-table title="Case details" [details]="vm.caseDetails"></app-details-table>
 
-  <app-details-table title="Hearing details" [details]="vm.hearingDetails"></app-details-table>
+  <app-details-table title="Request details" [details]="vm.hearingDetails"></app-details-table>
```

Comment:
Just noticed, this is request details, but the `[details]` are hearing details? Is that correct?

Reply:
the data is correct, but the variable could be better named. This change is incoming